### ISSUE TITLE
BackstopJS halting but nothing is appearing in the command line

### ISSUE THREAD

**Issue by LeeDownshift** [garris/BackstopJS#900]
I'm currently running some tests on a work project, and I appear to have hit a bit of an issue.  when running the backstop with puppeteer and with `"debugWindow": false` it seems to get so far through my scenarios and then decide its going to do nothing else. 

It doesn't crash in fact it took me a while to realise it had stopped as it was just sitting on the last step for a scenario and looked like it was just waiting for the next scenario to start. I've tried setting `"debug": true` but I don't appear to get any debugging output into the console.

What makes this problem even stranger is if I set `"debugWindow": true`, everything works fine (albeit the browser does take control of my machine for a while)

This is running on a rather large project and has somewhere in the region of 500 scenarios to work through. I have had it running on a project with around 215 scenarios and it works just fine.

Is there any way to get the command line output to be more verbose so that I can try and pinpoint the issue?

Thanks

**Comment by LeeDownshift** [comment]
Following on from this frustration, I decided to try running backstop using Chromy instead. So far I'm pleased to report that there doesn't appear to be the same halting behaviour without any warning. Chromy also seems to give a lot more in the way of debugging, however it runs much slower than puppeteer.

**Comment by garris** [comment]
If you are running 500 tests you may be having an issue with a single test or subset of tests which hang.  Have you tried isolating your scenarios to see which may be failing?

**Comment by LeeDownshift** [comment]
Yeah I thought that too, so I tried running just the test that it seemed to be halting on and it ran fine, I even tried running a subset that included the previously failing test and it still worked. 

Very strange, I definitely need to dig into this further but first I need to find a way of getting some debugging output from the tests...

**Comment by garris** [comment]
Did you try using `"debug": true` in the config?   If a test fails it should time out after 30 secs.     If 100 tests fail it could hold up processing for 5 mins (with an async limit of 10).   Still sounds like you just have a bunch of "waits" happening at the same time.
